### DISCUSSION TITLE
mpirun doesn't work on Linux machine - No error - but doesn't work

### DISCUSSION THREAD

**Discussion by MohsenSheibanian** [mpi4py/mpi4py#546] [Q&A]
I am trying to use it via `mpirun -n 8 python3 main.py --n_worker=8` but it doesn't work. I don't see any errors popping up! 
using `python3 main.py` use one single core and works well, but I need run the code faster! 
Attached is a list of the installed packages!
<img width="702" alt="Screenshot 2024-09-11 at 3 16 08 PM" src="https://github.com/user-attachments/assets/6dddeda1-c689-4b7d-a91e-e3c261d9eff9">

I'd appreciate it if someone can help me!

Thanks

**Comment by dalcinl** [comment]
This is most likely not an mpi4py issue. You should resubmit this issue to either conda-forge/mpi4py-feedstock or conda-forge/mpich-feedstock. First, you should check whether running a simple command like `mpirun -n 5 hostname` works as expected. If it does not, then the issue is definitely with MPICH, then the issue should be reported to conda-forge/mpich-feedstock.
